src/actions/categoriasActions.js

```diff
@@ -1,13 +1,12 @@
 import authAxios from "../config/authAxios";
 import { STATUS_OK } from "../config/constants";
-import response from "../config/network/response";
 
 export const getCategoryById = async (id) => {
```

Comment:
I think this function is searching products by category ID, and not searching categories by id. What about a name like:

"getProductsByCategory"

or similar?

Reply:
renamed